nexus/src/external_api/http_entrypoints.rs

```diff
@@ -337,14 +349,22 @@ async fn organizations_delete_organization(
 }
 
 /**
- * Update a specific organization.
+ * Update an organization
+ *
+ * Updates the specified organization by setting the values of the parameters passed.
+ * Any parameters not provided will be left unchanged.
  */
 /*
  * TODO-correctness: Is it valid for PUT to accept application/json that's a
  * subset of what the resource actually represents?  If not, is that a problem?
  * (HTTP may require that this be idempotent.)  If so, can we get around that
  * having this be a slightly different content-type (e.g.,
  * "application/json-patch")?  We should see what other APIs do.
+ * Other APIS work like the docs above, so it's fine. As any any parameters not passed are left
+ * unchanged.
+ * Leaving this comment here just to ping dave or whoever wrote it to let them know.
+ * Then I will delete.
+ * - Jess
```

Comment:
@davepacheco @ahl before I deleted the comment just wanted to let you know, you are correct and this is fine behavior lots of APIs do this

Reply:
So I was going to make a point of cleaning up these comments everywhere, if that's cool